docs/self-managed/setup/deploy/other/docker.md

```diff
@@ -189,7 +189,6 @@ docker run --rm --name=MyConnectorsInstance \
   -e CAMUNDA_CONNECTOR_POLLING_ENABLED=false \
   -e CAMUNDA_CONNECTOR_WEBHOOK_ENABLED=false \
```

Comment:
Wondering if these are still needed.

Reply:
Yes, these are still needed if you want to disable Operate connection without breaking anything else. Thanks for checking 🙂